### ISSUE TITLE
[Feat] Drive: confirm delete button

### ISSUE THREAD

**Issue by lambdagg** [mirukucafe/nyankey#8]
Adding a 'confirm delete' dialog in the Drive tab would be a nice feature to have, with some keybind to delete it without prompt (like SHIFT on Discord)

**Comment by initmd** [comment]
Maybe you could try using [AIScript](https://wiki.misskey.io/ja/function/AiScript) in order to do that if you wish to, then I think i can push this for everyone

**Comment by lambdagg** [comment]
I really think introducing this feature as part of the actual client is better.

**Comment by initmd** [comment]
Your plan is to make something toggelable right?

**Comment by lambdagg** [comment]
Not really. Just a nice-to-have feature for 'inattentive' people that would delete stuff without double-checking, an additional prompt just like Discord does - bypassable using i.e. the SHIFT key. It doesn't need to be toggleable, and would fit right into the client.

**Comment by initmd** [comment]
> Not really. Just a nice-to-have feature for 'inattentive' people that would delete stuff without double-checking, an additional prompt just like Discord does - bypassable using i.e. the SHIFT key. It doesn't need to be toggleable, and would fit right into the client.

Do you think I could assign this to you?

**Comment by initmd** [comment]
Though I think it's better to have it togellable. Need to get @wait-what's opinion about it

**Comment by lambdagg** [comment]
> > Not really. Just a nice-to-have feature for 'inattentive' people that would delete stuff without double-checking, an additional prompt just like Discord does - bypassable using i.e. the SHIFT key. It doesn't need to be toggleable, and would fit right into the client.
> 
> Do you think I could assign this to you?

Sure although that may be held for just a little while.

Waiting for Wait What's update.

**Comment by initmd** [comment]
WaitWhat will be unavailable for quite some time so I think you could just start working on it right away. If we decide to make it un-togglable it will not be that much difficult I guess.
